base/complex.jl

```diff
@@ -1,14 +1,22 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
 """
-    Complex{T<:Real} <: Number
+    AbstractComplex <: Number
+
+Abstract supertype for complex numbers.
+"""
+abstract type AbstractComplex <: Number
+end
```

Comment:
Maybe put it on the line above? Having it on a separate line seems as if we might add fields which we can't.